main.go

```diff
@@ -136,6 +136,7 @@ func main() {
 
 	slackbot.Action = func(c *cli.Context) error {
 		behaviors := bot.Behaviors{
+			bot.NewAliasBehavior(store).Behavior(),
```

Comment:
Should I follow `<entity><verb>Behavior` naming like `KarmaTrackingBehavior`, e.g. `AliasTransformingBehavior`?